src/SDKs/Search/DataPlane/AzSearch.DataPlane.props

```diff
@@ -0,0 +1,10 @@
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <PackageTags>Microsoft Azure Search;Search</PackageTags>
+    <VersionPrefix>5.0.0-preview</VersionPrefix>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
```

Comment:
@brjohnstmsft looks like the bug has been fixed that allows you specify Fx Version common for projects.
In that case, we will apply target Fx version from a common file and at that point, this line will be deleted from here (of course you will be involved in the PR), just as an FYI

Reply:
Thanks!